src/routes/tutorial/[slug]/codemirror.css

```diff
@@ -53,61 +53,82 @@
 }
 
 .cm-editor .cm-selectionMatch {
-	background: rgba(0,0,0,0.2);
+	background: var(--selection-color);
```

Comment:
The similar applies here as well:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/29677552/232077454-73651240-7c75-4b6e-a3d3-3e4d5b6de231.png">

---

<img width="465" alt="image" src="https://user-images.githubusercontent.com/29677552/232079470-135cec0e-038e-4f0a-b069-054a811808d6.png">